program/find-area-of-a-cube/find_area_of_a_cube.go

```diff
@@ -0,0 +1,11 @@
+package main
+
+import "fmt"
+
+func main() {
+	var num int
+	fmt.Print("Enter the Number to find Cube = ")
+	fmt.Scanln(&num)
+	cube := num * num * num
```

Comment:
This doesn't calculate the area of the cube.

Reply:
ohh I'm sorry, I got confused, I'll make changes as soon as possible